airflow/providers/pinecone/hooks/pinecone.py

```diff
@@ -126,3 +133,212 @@ def upsert(
             show_progress=show_progress,
             **kwargs,
         )
+
+    @staticmethod
+    def create_index(
+        index_name: str,
+        dimension: int,
+        timeout: int | None = None,
+        index_type: str | None = "approximated",
+        metric: str | None = "cosine",
+        replicas: int | None = 1,
+        shards: int | None = 1,
+        pods: int | None = 1,
+        pod_type: str | None = "p1",
+        index_config: dict[str, str] | None = None,
+        metadata_config: dict[str, str] | None = None,
+        source_collection: str | None = "",
```

Comment:
```suggestion
        index_type: str | None = "approximated",
        metric: str | None = "cosine",
        replicas: int | None = 1,
        shards: int | None = 1,
        pods: int | None = 1,
        pod_type: str | None = "p1",
        index_config: dict[str, str] | None = None,
        metadata_config: dict[str, str] | None = None,
        source_collection: str | None = "",
        timeout: int | None = None,
```

nit to keep order same as docstring